onnx/defs/tensor/defs.cc

```diff
@@ -1602,40 +1602,104 @@ ONNX_OPERATOR_SET_SCHEMA(
             "Constrain input 'X' and output 'Y' to all tensor types.")
         .SetDoc(Upsample_ver10_doc)
         .TypeAndShapeInferenceFunction([](InferenceContext& ctx) {
-          resizeShapeInference(ctx);
+          resizeShapeInference(ctx, false);
         }));
 
-static const char* Resize_ver10_doc = R"DOC(
-Resize the input tensor.
+static const char* Resize_ver11_doc = R"DOC(
+Resize the input tensor. In general, it calculates every value in the output tensor as a weighted average of neighborhood (a.k.a. sampling locations) in the input tensor.
```

Comment:
```suggestion
Resize the input tensor. In general, it calculates every value in the output tensor as a weighted average of neighborhood (a.k.a. sampled locations) in the input tensor.
```

Reply:
"sampling locations" is from https://github.com/tensorflow/tensorflow/blob/3ae2c6691b7c6e0986d97b150c9283e5cc52c15f/tensorflow/core/kernels/resize_bicubic_op.cc#L98. Do you think "sampled locations" is natural?